tox.ini

```diff
@@ -0,0 +1,7 @@
+[tox]
+envlist = py38
+[testenv]
+deps = 
+    -rrequirements.txt
```

Comment:
looks like a typo here

Reply:
I don't think it is:
https://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints